src/games/stendhal/server/maps/quests/UnicornHornsForRer.java

```diff
@@ -0,0 +1,180 @@
+package games.stendhal.server.maps.quests;
+import games.stendhal.common.parser.Sentence;
+import games.stendhal.server.entity.npc.*;
+import games.stendhal.server.entity.npc.action.*;
+import games.stendhal.server.entity.npc.condition.*;
+import java.util.*;
+
```

Comment:
These imports aren't used & should be deleted:
- `games.stendhal.common.parser.Sentence`
- `games.stendhal.server.entity.npc.condition.*`
- `java.util.*`

Reply:
If you are using [Eclipse](https://eclipse.org/), just hit `Ctrl+Shift+O`.